test/parser/LiteralOrIriTest.cpp

```diff
@@ -278,3 +278,45 @@ TEST(LiteralOrIri, Hashing) {
   ad_utility::HashSet<LiteralOrIri> set{lit, iri};
   EXPECT_THAT(set, ::testing::UnorderedElementsAre(lit, iri));
 }
+
+// _______________________________________________________________________
```

Comment:
A test is missing that tests the error case in the mutable `getIri()` function when there actually is a literal (test coverage).